### ISSUE TITLE
Feature request: Functionality for dash.bezierVertex()

### ISSUE THREAD

**Issue by mheesakkers** [garciadelcastillo/-dashed-lines-for-processing-#1]
I would really like to connect several bezier curves and then add a .offset(value) to animate the line in and out. Just like you would do in After Effects or with SVG dash strokes. With only seperate dash.bezier() calls you don't get the right effect because the dashes don't line up when joined.

**Comment by garciadelcastillo** [comment]
Yup, `dash.bezierVertex()` and similar methods are on the _to-do_ list ;)
